.github/workflows/publish.yml

```diff
@@ -0,0 +1,46 @@
+name: Publish Docker image
+on:
+# Will only run when release is published. 
+  push:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+jobs:
+  publish-images:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Update Packages
+        run: sudo apt-get update -yqq --fix-missing
+
+      - name: Check out code
+        uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Java
```

Comment:
What version of java included in `ubuntu-20.04`?

Reply:
Java 11